browser-test/mock-server.js

```diff
@@ -0,0 +1,83 @@
+/**
```

Comment:
In #565 I moved it to `system-test/mock-server.js`, I believe this file is not needed anymore